tart2ms/tart2ms.py

```diff
@@ -12,6 +12,10 @@
 import json
 import h5py
 import dask
+from dask.diagnostics import ProgressBar
+#dask.config.set(scheduler='threads')  # overwrite default with threaded scheduler
+dask.config.set(scheduler='processes')  # overwrite default with threaded scheduler
+#dask.config.set(scheduler='synchronous')  # overwrite default with threaded scheduler
```

Comment:
This is mainly for debugging and profiling. It may be good to put in a mode switch for this. Processes at least gets around some of the GIL issues